parity-clib/python/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.so
```

Comment:
hrm, move this to https://github.com/paritytech/parity-ethereum/blob/master/.gitignore

Reply:
Already in there, removed from this file. Strange though, pretty sure I added this for a reason :thinking: